include/ofi_hmem.h

```diff
@@ -291,6 +291,11 @@ ssize_t ofi_copy_to_hmem_iov(enum fi_hmem_iface hmem_iface, uint64_t device,
 			     size_t hmem_iov_count, uint64_t hmem_iov_offset,
 			     const void *src, size_t size);
 
+int ofi_copy_to_hmem(enum fi_hmem_iface iface, uint64_t device,
+				   void *dest, const void *src, size_t size);
+int ofi_copy_from_hmem(enum fi_hmem_iface iface, uint64_t device,
+				     void *dest, const void *src, size_t size);
```

Comment:
Move this set of changes into a separate patch.  Just move the entire static inline here

Reply:
fixed (gave you 1 more than you asked for)